src/main/java/seedu/address/ui/DeckListPanel.java

```diff
@@ -0,0 +1,49 @@
+package seedu.address.ui;
+
+import java.util.logging.Logger;
+
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.scene.control.ListCell;
+import javafx.scene.control.ListView;
+import javafx.scene.layout.Region;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.model.deck.Deck;
+
+/**
+ * Panel containing the list of persons.
+ */
+public class DeckListPanel extends UiPart<Region> {
+    private static final String FXML = "DeckListPanel.fxml";
+    private final Logger logger = LogsCenter.getLogger(PersonListPanel.class);
```

Comment:
I think the getLogger takes in `DeckListPanel.class` instead?

Reply:
fixed!